lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -909,6 +912,18 @@ DINode *SPIRVToLLVMDbgTran::transFunctionDecl(const SPIRVExtInst *DebugInst) {
   return DIS;
 }
 
+MDNode *SPIRVToLLVMDbgTran::transEntryPoint(const SPIRVExtInst *DebugInst) {
+  if (!isNonSemanticDebugInfo(DebugInst->getExtSetKind()))
```

Comment:
Is it a possible scenario? If we believe it's not - better to convert to assertion.